extensions/binding/org.eclipse.smarthome.binding.mqttbroker/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,34 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: MqttBroker Binding
+Bundle-SymbolicName: org.eclipse.smarthome.binding.mqttbroker;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.9.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: ., lib/moquette-broker-0.10.jar, lib/netty-all-4.1.13.Final.jar
```

Comment:
Why outdated? moquette-broker-0.10 is two months old and an important part of the idea of this binding - for openHAB to provide its broker out-of-the-box 😍

Reply:
The line is outdated, not the library. Or do you see this jar anywhere in this PR? No, because it is part of https://github.com/eclipse/smarthome/pull/3910 instead :-)